packages/js-dapi-client/lib/BlockHeadersProvider/BlockHeadersReader.js

```diff
@@ -95,13 +140,14 @@ class BlockHeadersReader extends EventEmitter {
   }
 
   /**
-   * A HOC that returns a function to subscribe to block headers and chain locks
+   * A HOF that returns a function to subscribe to historical block headers and chain locks
```

Comment:
What is HOF? ))

Reply:
A higher-order function :)

Reply:
Googled a bit on the matter. 
Found that there's no convention for naming the HOFs, but only a suggestion to keep it simple. 
Removed "HOF" from the function name after all.